Gemfile.lock

```diff
@@ -264,17 +267,15 @@ GEM
       rack-protection (~> 1.4)
       tilt (~> 1.3, >= 1.3.4)
     spring (1.3.3)
-    sprockets (2.12.3)
-      hike (~> 1.2)
-      multi_json (~> 1.0)
+    sprockets (3.2.0)
       rack (~> 1.0)
-      tilt (~> 1.1, != 1.3.0)
-    sprockets-rails (3.0.0.beta1)
-      actionpack (>= 4.0)
-      activesupport (>= 4.0)
-      sprockets (~> 2.8)
-    sprockets-strict-mode (1.0)
-      sprockets
+    sprockets-es6 (0.6.2)
+      babel-transpiler
+      sprockets (>= 3.0.0)
+    sprockets-rails (2.3.2)
```

Comment:
Without above, `sprockets-rails` actually picks the stable version instead of beta.